charts/internal/calico/templates/kube-controller/deployment-calico-kube-controllers.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         gardener.cloud/role: system-component
     spec:
       nodeSelector:
-        beta.kubernetes.io/os: linux
+        kubernetes.io/os: linux
```

Comment:
Will this work on 1.10 clusters?

Reply:
1.10 ? still exists, unsure, will revert this if 1.10 is required to work as well

Reply:
reverted